src/Common/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -765,78 +766,139 @@ function saturation_adjustment(e_int::FT, ρ::FT, q_tot::FT) where {FT<:Real}
   end
 end
 
-
```

Comment:
Update documentation in line 723: 
> The residual `q.tot - q.liq - q.ice` is the vapor specific humidity.

To make clear where `q.liq` etc. come from.